gh_pages/_source/getting_started_doc.rst

```diff
@@ -0,0 +1,67 @@
+Getting Started
+===============
+
+Install ROS and Catkin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+`Install ROS Noetic <http://wiki.ros.org/noetic/Installation/Ubuntu>`_.
+
+It is easy to miss steps when going through the ROS installation tutorial. If you run into errors in the next few steps, a good place to start is to go back and make sure you have installed ROS correctly.
+
+Once you have ROS installed, make sure you have the most up to date packages: ::
+
+  rosdep update
+  sudo apt-get update
+  sudo apt-get dist-upgrade
+
+Install `catkin <http://wiki.ros.org/catkin>`_ the ROS build system: ::
+
+  sudo apt-get install ros-noetic-catkin python-catkin-tools
+
+Install `wstool <http://wiki.ros.org/wstool>`_ the ROS workspace management tool: ::
+
+  sudo apt-get install python-wstool
+
+Build Instructions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Create a workspace (in this case we'll name it `tesseract_ws`): ::
+
+  mkdir -p ~/tesseract_ws/src
+
+Move into source directory: ::
+
+  cd ~/tesseract_ws/src 
+
+Clone Tesseract Planning repository into your workspace: ::
+
+  git clone https://github.com/ros-industrial-consortium/tesseract_planning
+
+If you are using the Robot Operating System (ROS), you can also clone the Tesseract ROS repository: ::
+
+  git clone https://github.com/ros-industrial-consortium/tesseract_ros
+
+Clone the repositories in the dependencies.rosinstall file using wstool or some other method (e.g. manually git cloning them): ::
+
+  wstool init ~/tesseract_ws/src/ ~/tesseract_ws/src/tesseract_planning/dependencies.rosinstall
+
+Install dependencies: ::
+
+  rosdep install -y --from-paths ~/tesseract_ws/src --ignore-src --rosdistro noetic
+
+Build the workspace using catkin tools: ::
+
+  cd ~/tesseract_ws/
+  source /opt/ros/noetic/setup.bash
+  catkin build
+
+Source the catkin workspace: ::
+
+  source ~/tesseract_ws/devel/setup.bash
+
+NOTE: For noetic tesseract_ext is not required. Install the following dependencies: libbullet-dev, libbullet-extras-dev and ros-noetic-fcl. Taskflow can be install using the PPA below.
```

Comment:
It's not ROS Noetic that makes `tesseract_ext` not required, it's the fact that the packages in `tesseract_ext` are distributed as binaries on Ubuntu 20.04
```suggestion
> `tesseract_ext` is not required on Ubuntu 20.04 because binary distributions of those packages are available. `taskflow` is the only remaining binary-distributed dependency that must be installed manually. This package can be installed using the PPA below.
```

Reply:
I decided to remove this note and add Taskflow as a step in the installation instructions. I also decided to add a note at the top indicating that the targeted OS for the instructions is Ubuntu 20.04. We may want to create another page/tab/section for installation on 18.04